Source/ThirdParty/AngelScript/source/as_config.h

```diff
@@ -228,6 +236,9 @@
 // AS_SPARC
 // Define this for SPARC CPU family
 
+// AS_E2K
```

Comment:
Same applies here. This is so that the next person doing the 3rd-party lib upgrade would not accidentally overwrite them.